flights/berks_ceh_clients_flight.yaml

```diff
@@ -24,9 +25,6 @@ entityDefinitions:
       nc.PersonSuffix:
         type: "nc.PersonSuffix"
         column: "NameSuffix"
-      nc.SSN:
-        type: "nc.SSN"
-        column: "SSN"
       nc.PersonSex:
         type: "nc.PersonSex"
         column: "Gender" 
```

Comment:
Actually this needs to be changed to M and F. All person datasets are written that way, makes linking more accurate.